packages/components/src/Image/tests/chromatic/SvgImage.stories.tsx

```diff
@@ -0,0 +1,53 @@
+
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Inline } from "../../../layout/index.ts";
+import { NoResults } from "../../assets/index.ts";
+import { SvgImage } from "../../src/SvgImage.tsx";
+
+const meta = {
+    title: "Components/SvgImage",
+    component: SvgImage,
+    parameters: {
+        chromatic: {
+            delay: 100
+        }
+    }
+} as Meta<typeof SvgImage>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Stroke = {
+    render: () => (
```

Comment:
prefer the args synthax over the render function whenever possible


```
const meta = {
    title: "Components/SvgImage",
    component: SvgImage,
    parameters: {
        chromatic: {
            delay: 100
        }
    },
    args: {   <------------------ args on all stories
        src: NoResults,
        aria-label: "No Results"
    }
} satisfies Meta<typeof SvgImage>;


export const Stroke = {
    args: {
        stroke: "neutral"
    }
} satisfies Story;

export const Styling = {
    render: (args) => (   <-- This is a good place to use render
        <Inline>
            <SvgImage className="stroke-red" {...args} />
            <SvgImage style={{ stroke: "red" }} {...args} />
        </Inline>
    )
} satisfies Story;
```

Reply:
I won't comment on every .stories.tsx, but i would expect this for all the files you created please